timemachine/fe/bar.py

```diff
@@ -322,7 +322,7 @@ def pair_overlap_from_ukln(u_kln: NDArray) -> float:
 
     """
     u_kn, N_k = ukln_to_ukn(u_kln)
-    return pymbar.mbar.MBAR(u_kn, N_k).compute_overlap()["scalar"]  # type: ignore
+    return max(0, 2 * pymbar.MBAR(u_kn, N_k).compute_overlap()["matrix"][0, 1])  # type: ignore
```

Comment:
Might:
(1) clamp output to `0 <= output <= 1`, (since, on bad inputs, entries of the matrix can also exceed 1.0)
(2) display a warning and/or throw an error if `overlap > (1 + epsilon)` or `overlap < (0 - epsilon)`, (this probably only happens when the estimate is also unreliable)

Reply:
- clamped to `0 <= output <= 1`